runtime/near-vm-logic/src/logic.rs

```diff
@@ -2232,6 +2232,11 @@ impl<'a> VMLogic<'a> {
             logs,
         }
     }
+
+    pub fn add_contract_compile_fee(&mut self, code_len: u64) {
+      self.gas_counter.pay_per_byte(contract_compile_bytes, code_len).unwrap();
```

Comment:
Why do we `unwrap` here? What if it overflows or exceeds the prepaid gas?

Reply:
This was just my ignorance in rust shinning through.  I thought that since it returned a result we had to check it.  Should this function instead also pass the result?